test/uk/gov/hmrc/economiccrimelevyregistration/controllers/actions/DataRetrievalActionSpec.scala

```diff
@@ -68,11 +83,19 @@ class DataRetrievalActionSpec extends SpecBase {
           .thenReturn(Future(updatedRegistration))
         when(mockAppConfig.privateBetaAccessCodes).thenReturn(Seq(privateBetaAccessCode))
 
+        val info = RegistrationAdditionalInfo(
+          registration.internalId,
+          Some(random[Int]),
+          None
+        )
+
+        when(mockRegistrationAdditionalInfoService.get(any())(any())).thenReturn(Future.successful(Some(info)))
```

Comment:
can we add a negative case for when mockRegistrationAdditionalInfoService.get doesn't return a successful response